### PR TITLE
fix: adjust markdown link format

### PR DESCRIPTION
**Pull request by jame2408**

### CODE REVIEW

**Comment by jame2408** [comment]
and, update README.md file.

File: content/vcs-features/index.md [hunk 1]
```diff
@@ -161,13 +161,13 @@ Read more in [Game Changers - Google's Mondrian](/game-changers/index.html#googl
...
 
 接下來，對於非 Google 開發世界來說，GitHub（而不是 Git 本身）成為具有內建程式碼審查工具（和工作流程）的平台。同樣，這也是在提交之前（或至少在提交到主分支之前）進行的審查。
 
 還有其他的程式碼審查技術，例如：Crucible（Atlassian）、UpSource（JetBrains）、Gerrit、Phabricator，但關鍵在於將其整合到平台體驗中。GitLab 和 RhodeCode 正在成為新興平台。
 
-詳細了解請參閱「Game Changers - Google's Mondrian」（[https://www.example.com/game-changers/index.html#google-s-internal-devops-2006-onwards](https://www.example.com/game-changers/index.html#google-s-internal-devops-2006-onwards)）和「Game Changers - GitHub's Pull Requests」（[https://www.example.com/game-changers/index.html#github-s-entire-platform-2008-onwards](https://www.example.com/game-changers/index.html#github-s-entire-platform-2008-onwards)）。
+詳細了解請參閱「[Game Changers - Google's Mondrian](https://www.example.com/game-changers/index.html#google-s-internal-devops-2006-onwards)」和「[Game Changers - GitHub's Pull Requests](https://www.example.com/game-changers/index.html#github-s-entire-platform-2008-onwards)」。
```
Comment (MilesChou) [inline content/vcs-features/index.md]:
```suggestion
詳細了解請參閱 [Game Changers - Google's Mondrian](/game-changers/index.html#google-s-internal-devops-2006-onwards) 和 [Game Changers - GitHub's Pull Requests](/game-changers/index.html#github-s-entire-platform-2008-onwards)。
```

Comment (MilesChou) [reply content/vcs-features/index.md]: 關於標題的錨點，好像還可以再調一下

File: README.md [hunk 2]
```diff
@@ -38,4 +38,4 @@ Pull-Requests, of course, if you're not a committer.
 
 English contributions (expansions of the material) to the `master` branch
 
-Chinese translations to the `cn` branch. - should always reflect the English content, so we can use merge tracking keep the two in step.
+Chinese translations to the `cn` or `tw` branch. - should always reflect the English content, so we can use merge tracking keep the two in step.
```
Comment (MilesChou) [inline README.md]: 這個跟下面的修改拆兩個 PR 比較好，不同的事

Comment (jame2408) [reply README.md]: @MilesChou 已調整，此項已重發 PR。

File: content/vcs-features/index.md [hunk 3]
```diff
@@ -161,13 +161,13 @@ Read more in [Game Changers - Google's Mondrian](/game-changers/index.html#googl
...
 
 接下來，對於非 Google 開發世界來說，GitHub（而不是 Git 本身）成為具有內建程式碼審查工具（和工作流程）的平台。同樣，這也是在提交之前（或至少在提交到主分支之前）進行的審查。
 
 還有其他的程式碼審查技術，例如：Crucible（Atlassian）、UpSource（JetBrains）、Gerrit、Phabricator，但關鍵在於將其整合到平台體驗中。GitLab 和 RhodeCode 正在成為新興平台。
 
-詳細了解請參閱「Game Changers - Google's Mondrian」（[https://www.example.com/game-changers/index.html#google-s-internal-devops-2006-onwards](https://www.example.com/game-changers/index.html#google-s-internal-devops-2006-onwards)）和「Game Changers - GitHub's Pull Requests」（[https://www.example.com/game-changers/index.html#github-s-entire-platform-2008-onwards](https://www.example.com/game-changers/index.html#github-s-entire-platform-2008-onwards)）。
+詳細了解請參閱 [Game Changers - Google's Mondrian](/game-changers/index.html#google-s-internal-devops-2006-onwards) 和 [Game Changers - GitHub's Pull Requests](/game-changers/index.html#github-s-entire-platform-2008-onwards)。
```
Comment (tedmax100) [inline content/vcs-features/index.md:170]: If PR #10  are merged, the link to related anchor will be worked.

```suggestion
詳細了解請參閱 [Game Changers - Google Mondrian](/game-changers/index.html#google-s-internal-devops-2006-onwards) 和 [Game Changers - GitHub Pull Requests](/game-changers/index.html#githubs-entire-initial-platform-2008)。
```

Comment (jame2408) [reply content/vcs-features/index.md:170]: I understand that PR #10 adjusts the anchors, and I appreciate your help in adjusting the hyperlinks. However, I am curious why the text needs to be changed? In the original text, "Google's" and "GitHub's" are indeed used.

Comment (MilesChou) [reply content/vcs-features/index.md:170]: 我目前看不出錯，先 merge 完再來找錯好了

**Review by MilesChou — APPROVED** [review]

**Pull request merged**